docs/user/gcp/install_upi.md

```diff
@@ -26,29 +38,61 @@ $ openshift-install create install-config
 ? Pull Secret [? for help]
 ```
 
-Edit the resulting `openshift-install.yaml` to set `replicas` to 0 for the `compute` pool:
+### Empty the compute pool (optional)
 
-```console
-$ sed -i '1,/replicas: / s/replicas: .*/replicas: 0/' install-config.yaml
+If you do not want the cluster to provision compute machines, edit the resulting `install-config.yaml` to set `replicas` to 0 for the `compute` pool.
+
+```sh
+python -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.full_load(open(path));
+data["compute"][0]["replicas"] = 0;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
-Create manifests to get access to the control-plane Machines and compute MachineSets:
+### Create manifests
+
+Create manifest to enable customizations which are not exposed via the install configuration.
 
 ```console
 $ openshift-install create manifests
 INFO Consuming "Install Config" from target directory
 ```
 
-From the manifest assets, remove the control-plane Machines and the compute MachineSets:
+### Remove control plane machines
 
-```console
-$ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
-$ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+Remove the control plane machines from the manifests.
+We'll be providing those ourselves and don't want to involve [the machine-API operator][machine-api-operator].
+
+```sh
+rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
+```
+
+### Remove compute machinesets (Optional)
+
+If you do not want the cluster to provision compute machines, remove the compute machinesets from the manifests as well.
+
+```sh
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ```
 
-You are free to leave the compute MachineSets in if you want to create compute
-machines via the machine API, but if you do you may need to update the various
-references (`subnetwork`, etc.) to match your environment.
+### Remove DNS Zones (Optional)
+
+If you don't want [the ingress operator][ingress-operator] to create DNS records on your behalf, remove the `privateZone` and `publicZone` sections from the DNS configuration.
+If you do so, you'll need to [add ingress DNS records manually](#add-the-ingress-dns-records) later on.
```

Comment:
You need an `-optional` or some such to match your slightly-different header for the later section.

Reply:
Ah... I missed that. Fixed.